### PR TITLE
Various UI fixes

### PR DESCRIPTION
**Pull request by markhiveit**

### CODE REVIEW

**Comment by markhiveit** [comment]
Seems the tests are still failing to add meta guidance content text to subjects intermittently *only* on the pipeline. I'll give it a look tomorrow.

File: tests/robot-tests/tests/libs/admin-common.robot [hunk 1]
```diff
@@ -269,9 +269,12 @@ user deletes editable accordion section content block
...
-    [Arguments]   ${accordion_heading}
+user enters text into meta guidance data file content editor
+    [Arguments]   ${accordion_heading}   ${text}
     ${accordion}=  user gets accordion section content element  ${accordion_heading}
     user waits until parent contains element  ${accordion}   xpath://*[@data-testid="Content"]//*[@role="textbox"]
     ${editor}=  get child element  ${accordion}  xpath://*[@data-testid="Content"]//*[@role="textbox"]
-    [Return]  ${editor}
+    user checks page does not contain testid   fileGuidanceContent-focused
+    user clicks element  ${editor}
+    user waits until page contains testid   fileGuidanceContent-focused
```
Comment (markhiveit) [inline tests/robot-tests/tests/libs/admin-common.robot:279]: I did a lot of umming and ahhing about how to wait for the content editor to be ready to receive text. In the end, I decided this was the best option.

Comment (ntsim) [reply tests/robot-tests/tests/libs/admin-common.robot:279]: Has there been problems with the editor being ready to receive text? Running it locally, it seems to run fine

Comment (markhiveit) [reply tests/robot-tests/tests/libs/admin-common.robot:279]: It's been failing intermittently - I'm not 100%, but I think it's only been happening on the pipeline. Anyway, I'm hoping this fixes it.

Comment (ntsim) [reply tests/robot-tests/tests/libs/admin-common.robot:279]: Cool, we'll go with it then. Wish the pipeline wouldn't randomly fail on stuff like this though!

File: tests/robot-tests/tests/libs/admin-common.robot [hunk 2]
```diff
@@ -269,9 +269,12 @@ user deletes editable accordion section content block
     user clicks button  Remove block  ${block}
     user clicks button  Confirm
 
-user gets meta guidance data file content editor
-    [Arguments]   ${accordion_heading}
+user enters text into meta guidance data file content editor
```
Comment (ntsim) [inline tests/robot-tests/tests/libs/admin-common.robot:272]: Think it'd be good to have the original `user gets meta guidance data file content editor` keyword as well. In `publish_release_and_amend` I've actually used this to check the editor has been populated with the correct text content.

**Review by ntsim — CHANGES_REQUESTED** [review]

File: tests/robot-tests/tests/libs/admin-common.robot (hunk 1 above)
Comment (markhiveit) [reply tests/robot-tests/tests/libs/admin-common.robot:279]: I have wondered if we had laptops that are the same specs as the Azure agents we might be more likely to recreate this stuff locally - almost always pipeline only failures seem to be timing related.

Comment (ntsim) [reply tests/robot-tests/tests/libs/admin-common.robot:284]: Hah good luck running all of EES on just 2 cores!

**Review by ntsim — APPROVED** [review]
Approved but seems like there's merge conflicts

**Pull request merged**